serialization/items/contentItems/location/7IsPs29p9UmIJxJTX5OxyQ.contentItem.yaml

```diff
@@ -0,0 +1,18 @@
+﻿id: 7IsPs29p9UmIJxJTX5OxyQ
+name: America - HQ
+fields:
+  title:
+    value: America - HQ
+    type: ShortText
+  address:
+    value: Why Worry Lane
+    type: ShortText
+  city:
+    value: Arizona city
+    type: ShortText
+  country:
+    value: USA
+    type: ShortText
+  phone:
+    value: 044 773663
```

Comment:
I have updated this phone number to look more like a real USA phone number on Content Hub ONE. I also changed "Play!" for "PLAY!". Could you please update pull the additional items, and commit them before merging? Thanks